apps/ledger-live-mobile/fastlane/Fastfile

```diff
@@ -279,9 +297,8 @@ platform :ios do
       begin
         pilot(
           skip_submission: false,
-          app_identifier: "com.ledger.live",
+          app_identifier: ENV["APP_IDENTIFIER"],
```

Comment:
i like this one 👍

Reply:
Actually I overlooked this one - this needs to adopt the pattern above

Reply:
Gah - sorry ignore the comment above - this is inside the `if (options[:nightly])`, so we don't need to change it. The else clause below contains the hardcoded app identifier for all other release types.

IMO this Fastfile is quite confusing and could be cleaned up significantly, but I just want to be deliberate about what we are taking on in this PR.